modules/testing/pages/integration/canceling.adoc

```diff
@@ -0,0 +1,14 @@
+= Canceling Integration pipelineruns for Pull Requests
+
+Pushing a new change to your PR creates a new build pipelinerun, followed by a new snapshot if build succeeded, then integration tests are triggered.
+In case you happen to push another change in your Pull Request, while integration tests pipelinerun are still running, the previous pipelinerun will be canceled.
```

Comment:
```suggestion
In case you happen to push another change in your Pull Request, while integration tests pipelinerun are still running, the previous test pipelinerun will be canceled.
```